lib/stream/filter_slim_graph.js

```diff
@@ -0,0 +1,45 @@
+var keys = require("lodash/keys");
+var omit = require("lodash/omit");
+var through = require("through2");
+var assign = require("lodash/assign");
+var includes = require("lodash/includes");
+
+module.exports = function() {
+	return through.obj(function(data, enc, done) {
+		try {
+			done(null, filterGraph(data));
+		} catch (err) {
+			done(err);
+		}
+	});
+};
+
+var blackList = [
+	"npm",
+	"npm-convert",
+	"npm-crawl",
+	"npm-extension",
+	"npm-load",
+	"npm-utils",
+	"semver",
+	"@dev"
+];
```

Comment:
sounds good, I kept the blacklist logic but instead of statically defining it I built it from the configMain node deps as suggested https://github.com/stealjs/steal-tools/pull/731/commits/47c6ace721d0c0abc5f8c4221566c091c118ee54